tests/subtype/test_qc.py

```diff
@@ -48,7 +47,7 @@ def test_intermediate_subtype():
                  avg_tile_coverage=37.04102564102564,
                  qc_status=None,
                  qc_message=None)
-    df = pd.read_csv('tests/data/se_intermediate_subtype_df.csv')
+    df = pd.read_csv('tests/data/subtype/se_intermediate_subtype_df.csv')
     p = init_subtyping_params(args=None, scheme=scheme)
```

Comment:
just a small thing, but perhaps a more descriptive variable name for p?

Reply:
Changed in 9511966